src/main/java/be/seeseemelk/mockbukkit/ServerMock.java

```diff
@@ -423,7 +423,7 @@ public String getName()
 	@Override
 	public String getVersion()
 	{
-		return "0.1.0";
+		return getBukkitVersion() + "-MockBukkit";
```

Comment:
I'm not sure if it is safe to add text to this all willy-nilly. Some plugins and frameworks (e.g.: Paper) perform regex matches on this. @TheBusyBiscuit probably knows what version strings can actually be found in the wild. We should try to conform to that format as much as possible.

Reply:
I think I might revert this change and open another issue/pull request to address this as it's just indirectly related to this topic. (WorldEdit doesn't loading cause it thinks the server version is too old was the origin)

Reply:
👍🏻 Feel free to open a new Pull Request for this and I will merge it asap.
Since the library PaperLib also relies on the version following a certain format:
https://github.com/PaperMC/PaperLib/blob/master/src/main/java/io/papermc/lib/environments/Environment.java#L45-L68

And many plugins utilise this library, so following that (unspoken) convention would be nice.
"MockBukkit (MC: ${bukkitVersion})" would probably be ideal.